account.py

```diff
@@ -102,6 +102,9 @@ def check_owners(cls, accounts):
                         ], limit=1)
                 for record in records:
                     target = record.account_bank_from
+                    if (not target or (record.payment_type
```

Comment:
account_bank_from get party from payment type.
- from party in case payment type account_bank is "party"
- from company in case payment type account_bank is "company"
- from party payment type, in case payment type account_banl is "other".